store/tikv/oracle/oracles/pd.go

```diff
@@ -35,8 +35,9 @@ const slowDist = 30 * time.Millisecond
 type pdOracle struct {
 	c pd.Client
 	// txn_scope (string) -> lastTSPointer (*uint64)
-	lastTSMap sync.Map
-	quit      chan struct{}
+	lastTSMap            sync.Map
+	lastArrivalTSPointer *uint64
```

Comment:
```suggestion
	lastArrivalTS uint64
```

Reply:
Thanks for your review, I agree with you.